examples/spmd/Makefile

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) 2019, University of Washington All rights reserved.
+# 
+# Redistribution and use in source and binary forms, with or without modification,
```

Comment:
This file is an amalgamation/move of the files tests.mk, and rules.mk and Makefile in regression/spmd/